learning_resources/etl/loaders.py

```diff
@@ -347,7 +348,7 @@ def upsert_course_or_program(
         ).order_by("-updated_on")
         if existing_courses.count() > 1:
             for course in existing_courses[1:]:
-                course.delete()
+                resource_delete_actions(course)
```

Comment:
`resource_delete_actions` doesn't actually delete the resource, right—it just calls the associated plugins? Shouldn't we do both?

Reply:
it does do both, via the associated plugin function: https://github.com/mitodl/mit-open/blob/main/learning_resources_search/plugins.py#L143-L148

```python
    @hookimpl
    def resource_delete(self, resource):
        """
        Remove a resource from the search index and then delete the object
        """
        self.resource_unpublished(resource)
        resource.delete()
```